libs/CMakeLists.txt

```diff
@@ -270,7 +290,7 @@ import_external_library(protobuf STATIC
   CONFIGURE_COMMAND true)
 
 import_external_library(pulsar STATIC
-  ${CMAKE_CURRENT_SOURCE_DIR}/pulsar/pulsar-client-cpp/lib/libpulsarwithdeps.a
+  ${CMAKE_CURRENT_SOURCE_DIR}/pulsar/install/lib/libpulsar.a
```

Comment:
Yeah, when using Ninja it complained about duplicate symbols around zstd. Hence I removed `libpulsarwithdeps.a` and just use `libpulsar.a` and added Snappy explicitly.
Its the snappy part...not sure what provides it as build + then also when packaged in docker etc.

Reply:
Snappy is in the toolchain -> that might be a problem for packaging because that has to be installed manually in the `memgraph.deb/rpm` case (for each operating system). But again, I think we have to make the overhaul of the streaming clients... Not sure what's the best, probably to somehow make it work with `libpulsarwithdeps.a` because that won't break packaging (e2e and packaging are both important). Some hacky solution is also ok temporarily, within v2.20 we should make the streaming overhaul primarily because of SASL support.